src/less/multi-avatars.less

```diff
@@ -0,0 +1,32 @@
+.multi-avatars {
+  display: inline-flex;
+  flex-direction: row-reverse;
+  width: 40px;
+  height: 40px;
+  position: relative;
+  margin-right: 0.8rem;
+}
+
+.multi-avatars__background-avatar {
+  position: absolute;
+  right: 0;
+  top: 0;
+  border: 2.0px solid #fff;
+  border-radius: 50%;
```

Comment:
if you are already using the avatar class here, you probably don't need `border-radius: 50%` since that class has that. Same for `overflow` property. It wouldn't hurt to check against the existing classes to make sure we're not adding excess properties to these new classes.

Reply:
👍 good catch